### ISSUE TITLE
Error message when uploading Field Report

### ISSUE THREAD

**Issue by mmusori** [IFRCGo/go-frontend#555]
## Issue

When a field report was created  for Nigeria cholera outbreak the following  error message was returned after clicking the   ``save and continue`` button ``could not submit field report``.  But the field report had been uploaded. The team then tried again and obtained another error message even though the field report had been uploaded a second time. 
![image](https://user-images.githubusercontent.com/7780809/50630197-2be0ff00-0f40-11e9-9108-8405a367c5df.png)

## Steps to reproduce

*If you can reliably reproduce the issue, please list the steps taken to do so.*

## Expected behavior

*Please describe how this feature should be working.*

**Comment by szabozoltan69** [comment]
Please give me more details. I need all the data you entered on the previous input pages, so that this issue should be reproduced.
A possible root of the error can be the broken link of some document referred.

**Comment by mmusori** [comment]
All the information submitted in the field report is available on: https://go.ifrc.org/reports/11638
The report was submitted but they still got an error message.

**Comment by szabozoltan69** [comment]
Thank you. And all the needed data is saved on the existing field report https://go.ifrc.org/reports/11638 ? Nothing is missing?

**Comment by mmusori** [comment]
Yes, that is the first field report they submitted then they got an error message so they submitted a second one:  https://go.ifrc.org/reports/11639
And got an error message as well. 
But on another note the Field Report is supposed to create an emergency and no emergency was created on the 30th of December. 
![image](https://user-images.githubusercontent.com/7780809/50634226-3b674480-0f4e-11e9-96da-6f4ca2ed1e16.png)

**Comment by szabozoltan69** [comment]
Well, the "Emergency" field is a (typing-time) loopkup field on field report creation so I do not think that emergency should be created on-the-fly when creating a field report.

I suggest to create the Emergency before, and then create the field report. (Let's clear this sequence question before continuing the error investigation.)

**Comment by mmusori** [comment]
@Zoltan for GO Phase 2 we decided that Emergencies are created by a field report or a GDACS alert. So the creation of a field report should automatically trigger the creation of an emergency. 
Also thanks for resolving the Nigeria Cholera issue.

**Comment by szabozoltan69** [comment]
Thank you @mmusori
